pom.xml

```diff
@@ -99,6 +99,7 @@
     <google.http-client.version>1.39.2-sp.1</google.http-client.version>
     <google.oauth-client.version>1.31.5</google.oauth-client.version>
     <google.protobuf.version>3.17.3</google.protobuf.version>
+    <google.cloud.storage.grpc.version>2.0.1-alpha</google.cloud.storage.grpc.version>
```

Comment:
Move it after `google.cloud-bigquerystorage.version` and rename to `google.cloud-storage.version` for consistency